README.md

```diff
@@ -134,6 +135,70 @@ import "@asyncapi/react-component/styles/default.css";
 import "@asyncapi/react-component/styles/default.min.css";
 ```
 
+### Custom styles
+The AsyncAPI React component does not set any global fonts and any global reset styles. This provides the option to define your custom `font-family` and other styling.
```

Comment:
```suggestion
### Custom styles
The AsyncAPI React component does not set any global fonts or reset styles. This component provides the option to define your custom `font-family` and other styling.
```

Reply:
`The AsyncAPI React component does not set any global fonts and any global reset styles`
The "or" would not be sufficient since it there are no global fonts sets and a number of styles are reset.

Reply:
I updated that part.